rclcpp/include/rclcpp/macros.hpp

```diff
@@ -31,11 +31,11 @@
 #define RCLCPP_MAKE_SHARED_DEFINITIONS(Class) \
   typedef std::shared_ptr<Class> SharedPtr; \
   \
-  template<typename...Args> \
+  template<typename ... Args> \
   static std::shared_ptr<Class> \
-  make_shared(Args &&...args) \
+  make_shared(Args && ... args) \
   { \
-    return std::make_shared<Class>(std::forward<Args>(args)...); \
+    return std::make_shared<Class>(std::forward<Args>(args) ...); \
```

Comment:
> Can you please test OS X / Windows?

We should run the build farm over all of it, so I won't test this individual item on each system. I think that if it compiles it will be fine.